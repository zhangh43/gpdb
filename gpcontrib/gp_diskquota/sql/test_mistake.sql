-- to make sure that the schema 'notfoundns' is really not found
select nspname from pg_namespace where nspname = 'notfoundns';
select diskquota.set_schema_quota('notfoundns', '1 MB');
