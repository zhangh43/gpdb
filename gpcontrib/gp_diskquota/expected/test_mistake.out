-- to make sure that the schema 'notfoundns' is really not found
select nspname from pg_namespace where nspname = 'notfoundns';
 nspname 
---------
(0 rows)

select diskquota.set_schema_quota('notfoundns', '1 MB');
ERROR:  schema "notfoundns" does not exist
